### PR TITLE
Add Gapcloser Event

### PR DESCRIPTION
**Pull request by NightMoon032**
Maybe someone need to add Interrupt Spell Event

### CODE REVIEW

File: Aimtec.SDK/Events/Gapcloser.cs [hunk 1]
```diff
@@ -0,0 +1,1062 @@
...
+            unit.SpellName = Args.SpellData.Name;
+            unit.StartPosition = Args.Start;
+            unit.EndPosition = Args.End;
+            unit.StartTick = Game.TickCount;
+            unit.HaveShield = HaveShiledBuff(sender);
+        }
+
+        private static bool HaveShiledBuff(Obj_AI_Base target)
+        {
+            if (target == null || target.IsDead || target.Health <= 0 || !target.IsValidTarget())
```
Comment (LHS1140b) [inline Aimtec.SDK/Events/Gapcloser.cs]: `target.IsDead || target.Health <= 0` would `target.IsValidTarget()` not be enough here?

it checks

```
if (target == null || !target.IsValid || target.IsDead || target.IsInvulnerable || !target.IsVisible || !target.IsTargetable)
            {
                return false;
            }
```

**Comment by LHS1140b** [comment]
@NightMoon032 why did u closed it

**Comment by LHS1140b** [comment]
well besides the orbwalk changes later on

**Comment by NightMoon032** [comment]
close it
have my new idea for this update

**Comment by LHS1140b** [comment]
@NightMoon032 Okay thank you

**Comment by A447115172** [comment]
兄弟 Sharpaio 在哪里哦 找不到

**Pull request closed without merging**